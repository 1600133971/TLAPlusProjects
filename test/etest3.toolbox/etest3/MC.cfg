\* INIT definition
INIT
Init
\* NEXT definition
NEXT
Next
\* Generated on Sun Dec 13 19:43:51 CST 2020