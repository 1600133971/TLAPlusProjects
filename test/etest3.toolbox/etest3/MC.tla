---- MODULE MC ----
EXTENDS etest3, TLC

=============================================================================
\* Modification History
\* Created Sun Dec 13 19:43:51 CST 2020 by Administrator
