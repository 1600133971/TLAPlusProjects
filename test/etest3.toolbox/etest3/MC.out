@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.15 of Day Month 20?? (rev: 68d4252)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2187:0 @!@!@
Running breadth-first search Model-Checking with fp 31 and seed -7985172571126072374 with 6 workers on 12 cores with 1211MB heap and 2721MB offheap memory [pid: 10768] (Windows 10 10.0 amd64, AdoptOpenJDK 14.0.1 x86_64, OffHeapDiskFPSet, DiskStateQueue).
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file D:\tla+projects\etest3.toolbox\etest3\MC.tla
Parsing file D:\tla+projects\etest3.toolbox\etest3\etest3.tla
Parsing file C:\toolbox\plugins\org.lamport.tlatools_1.0.0.202012110300\tla2sany\StandardModules\TLC.tla
Parsing file C:\toolbox\plugins\org.lamport.tlatools_1.0.0.202012110300\tla2sany\StandardModules\Naturals.tla
Parsing file C:\toolbox\plugins\org.lamport.tlatools_1.0.0.202012110300\tla2sany\StandardModules\Sequences.tla
Parsing file C:\cygwin\usr\local\lib\tlaps\FiniteSets.tla
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module FiniteSets
Semantic processing of module TLC
Semantic processing of module etest3
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2020-12-13 19:43:52)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
"This test should deadlock after finding two states"  TRUE
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 1 distinct state generated at 2020-12-13 19:43:52.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 2114:1 @!@!@
Deadlock reached.
@!@!@ENDMSG 2114 @!@!@
"This test should deadlock after finding two states"  TRUE
@!@!@STARTMSG 2121:1 @!@!@
The behavior up to this point is:
@!@!@ENDMSG 2121 @!@!@
@!@!@STARTMSG 2217:4 @!@!@
1: <Initial predicate>
/\ x = {}
/\ y = {}
/\ z = 1

@!@!@ENDMSG 2217 @!@!@
@!@!@STARTMSG 2217:4 @!@!@
2: <Next line 13, col 9 to line 17, col 18 of module etest3>
/\ x = {1}
/\ y = {}
/\ z = 1

@!@!@ENDMSG 2217 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics at 2020-12-13 19:43:52
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2773:0 @!@!@
<Init line 8, col 1 to line 8, col 4 of module etest3>: 2:2
@!@!@ENDMSG 2773 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 8, col 9 to line 11, col 16 of module etest3: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Next line 13, col 1 to line 13, col 4 of module etest3>: 1:2
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 13, col 12 to line 13, col 17 of module etest3: 5
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 13, col 12 to line 13, col 12 of module etest3: 3
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 13, col 16 to line 13, col 17 of module etest3: 3
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 14, col 12 to line 14, col 19 of module etest3: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 15, col 12 to line 15, col 27 of module etest3: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 16, col 12 to line 16, col 17 of module etest3: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 16, col 17 to line 16, col 17 of module etest3: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 17, col 12 to line 17, col 18 of module etest3: 4
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(2) at 2020-12-13 19:43:52: 2 states generated (169 s/min), 2 distinct states found (169 ds/min), 0 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
2 states generated, 2 distinct states found, 0 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2194:0 @!@!@
The depth of the complete state graph search is 2.
@!@!@ENDMSG 2194 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 712ms at (2020-12-13 19:43:52)
@!@!@ENDMSG 2186 @!@!@
